examples/github-stars-example/README.md

```diff
@@ -0,0 +1,307 @@
+# Overview
+
+In this example we will use Versatile Data Kit to create Streamlit web app that displays GitHub Star History:
+
+[<img src="https://user-images.githubusercontent.com/11227374/192750161-dfca62f0-5ea3-4300-8887-4413c8dbbf05.png" width="500"/>](Github)
+
+The job will insert the data from GitHub API into local sqlite table and create a Streamlit app. It also edits the default streamlit color schema.
+
+Before you continue, make sure you are familiar with the
+[Getting Started](https://github.com/vmware/versatile-data-kit/wiki/Getting-Started) section of the wiki.
+
+How to run this job locally
+-----
+To run this job, follow the steps:
+
+1. Clone the repo
+```
+git clone https://github.com/vmware/versatile-data-kit.git
+```
+2. Install vdk
+```
+pip install -U pip setuptools wheel
+pip install quickstart-vdk
+```
+3. Now do vdk create, enter the job name, team and location
+```
+vdk create
+
+Job Name: local-github-stars
+Job Team: test
+Path to where sample data job will be created locally [/home/gita/vdk]:
+Data Job with name local-github-stars created locally in /home/gita/vdk/local-github-stars.
+```
+4. Clear the new directory and copy example files to it
+```
+rm local-github-stars/*
+cp versatile-data-kit/examples/github-stars-example/github-stars-example-job/* local-github-stars/
```

Comment:
why don't they just run it from the examples folder? 
what is the advantage of copying it somewhere else?

Reply:
If I remove that part, I can also remove this. Maybe second opinion could be good.

Reply:
Ah I see!

Reply:
As I mentioned in the other comment let users use CLI command `vdk properteis` and they should not modify the source code.

Reply:
I wish to keep this part for  2 reasons: 
1. to show how easy it is to create a job
2. to keep the structure of the file.

I'll remove this if you find it necessary